### PR TITLE
Update lessons links from 21 to 22 in pegasus

### PR DESCRIPTION
**Pull request by davidsbailey**
Continues [PLAT-1861](https://codedotorg.atlassian.net/browse/PLAT-1861). Copies approach from https://github.com/code-dot-org/code-dot-org/pull/47003:
* 1st commit updates links to lessons which have moved
* 2nd commit updates links to lessons which have NOT moved
* some links point to lessons which have been removed, or otherwise need content editor attention. these have been deferred as future work.

## Testing story

Manually verified that old and new links appear the same. results were tracked in the [spreadsheet](https://docs.google.com/spreadsheets/d/139bb4ID2IBFf-6491I_JfM3CIP36WA6BAy_YQc1uing/edit#gid=1653116251).

## Deployment strategy

the plan is to ship it today! sorry for the rush 😬 

## Future work

curriculum input on updating these entries from the spreadsheet:

./curriculum/unplugged.md.partial | https://studio.code.org/s/coursef-2021/lessons/13
-- | --

./educate/spritelab.haml | https://studio.code.org/s/coursee-2021/lessons/3/levels/3
-- | --

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/educate/csc.md.partial [hunk 1]
```diff
@@ -32,7 +32,7 @@ Continue making connections between computer science and traditional subjects wi
...
 
-{{ tutorial_promo2_single_course, heading: "", background_image: "/images/fit-1940/csc/helloworld/helloworld_textbox.jpg", text_color: "white", grades: "3+", subject: "All", duration: "1 hour", box_background_color: "rgb(0,0,0,0.4)", right_heading: "Hello World", right_description: "Get students started with the basics of Sprite Lab. With these new skills, students can create projects on any topic, including some of the cross-curricular options below.", right_specs: "", right_button_url: "https://studio.code.org/s/hello-world-food-2021/lessons/1", right_button_text: "View lesson plan" }}
+{{ tutorial_promo2_single_course, heading: "", background_image: "/images/fit-1940/csc/helloworld/helloworld_textbox.jpg", text_color: "white", grades: "3+", subject: "All", duration: "1 hour", box_background_color: "rgb(0,0,0,0.4)", right_heading: "Hello World", right_description: "Get students started with the basics of Sprite Lab. With these new skills, students can create projects on any topic, including some of the cross-curricular options below.", right_specs: "", right_button_url: "https://studio.code.org/s/hello-world-food-2022/lessons/1", right_button_text: "View lesson plan" }}
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/public/educate/csc.md.partial]: @davidsbailey I don't think hello-world-food-2022 exists

Comment (dmcavoy) [reply pegasus/sites.v3/code.org/public/educate/csc.md.partial]: same with the other csc links in this file that were updated

Comment (davidsbailey) [reply pegasus/sites.v3/code.org/public/educate/csc.md.partial]: great catch Dani! I did check these when I went through the spreadsheet, but I did not do what my spreadsheet said to do for these, which was the leave the url how it was. thank you for the careful review!

**Review by dmcavoy — APPROVED** [review]
Good to go once the CSC links are set back. Thanks!

**Comment by davidsbailey** [comment]
drone ui test failures look unrelated

**Pull request merged**